### PR TITLE
Add documentation for tomorrowio as well as climacell deprecation

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
Because ClimaCell has rebranded, we are creating a brand new integration and automatically migrating climacell users over to it. Under the hood the code is mostly the same, so the new tomorrowio page is mostly a copy/paste from the climacell docs. I also added a note on the climacell page about deprecation and removed installation instructions since it will no longer be available for setup.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68156
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2742
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/climacell.markdown [hunk 1]
```diff
@@ -17,13 +17,11 @@ ha_platforms:
   - weather
 ---
 
-The ClimaCell integration allows you to obtain weather, air quality, pollen, and fire information from the [ClimaCell API](https://www.climacell.co/weather-api/).
-
-## Obtain an API key
-
-You can obtain a free API key by signing up with [ClimaCell](https://developer.climacell.co/sign-up?_ga=2.137889264.1908484805.1591592950-510691096.1591288729).
+<div class='note warning'>
```
Comment (frenck) [inline source/_integrations/climacell.markdown]: I think we should just add a redirect instead (`source/_redirects`).

If you do so, please be sure to rebase this PR before doing that, as the redirect file has massively changed this week.

File: source/_integrations/tomorrowio.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - weather
+---
+
+The Tomorrow.io integration allows you to obtain weather, air quality, pollen, and fire information from the [Tomorrow.io API](https://www.tomorrow.io/weather-api/).
+
+## Obtain an API key
+
+You can obtain a free API key by signing up with [Tomorrow.io](https://www.tomorrow.io/weather-api/).
+
+The integration will automatically set the refresh interval based on the number of Tomorrow.io integrations that are using the same API key for a given Home Assistant instance. The integration currently assumes you are using a free account so the max requests per day is 500 (the refresh interval is calculated such that you should only use up around 90% of the quota). If you want to use a paid account to increase the number of max requests and increase the update frequency, please open a Home Assistant issue for the integration.
```
Comment (frenck) [inline source/_integrations/tomorrowio.markdown]: > If you want to use a paid account to increase the number of max requests and increase the update frequency, please open a Home Assistant issue for the integration.

? So it is possible? It isn't possible? Its a bug and you want people to report it? 😬

File: source/_integrations/tomorrowio.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Supported Forecast Types
+
+| Forecast Type | Description                                                                                                      |
+|---------------|------------------------------------------------------------------------------------------------------------------|
+| `nowcast`     | Up to the 1 minute predictions. Supports 300 minutes or a max of 30 forecasts depending on the chosen `timestep` |
+| `hourly`      | Hourly forecasts for the next 24 hours                                                                           |
+| `daily`       | Daily  forecasts for the next 14 days                                                                            |
```
Comment (frenck) [inline source/_integrations/tomorrowio.markdown]:
```suggestion
| `daily`       | Daily forecasts for the next 14 days                                                                            |
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tomorrowio.markdown (hunk 2 above)
Comment (raman325) [reply source/_integrations/tomorrowio.markdown]: lol fair enough, I will remove that

**Review by frenck — APPROVED** [review]
Thanks, @raman325 👍

**Pull request merged**